styles/index.js

```diff
@@ -0,0 +1,5 @@
+import Devices from 'react-native-device-detection'
+import tablet from './tablet'
+import mobile from './mobile'
+
+export default Devices.isTablet ? tablet : mobile
```

Comment:
this looks like the right approach. So all we need to do is to have the same style name on both tablet.js and mobile.js, then apply the style on each screen and components without additional isTablet check all over the place.

Reply:
kk